form_dodaj.php

```diff
@@ -0,0 +1,51 @@
+
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+<b>Dodaj lek!</b><br><br>
+
+<form action="dodaj.php" method="post">
+    <label>Nazwa leku <input name="lek" type="text"></label>
+    <?php
+    if (isset($_SESSION['e_lek']))
```

Comment:
Dlaczego korzystasz z sesji? Jaki problem chcesz rozwiązać?

Reply:
Gdzie ustawiasz te wszystkie zmienne sesyjne, np. $_SESSION['e_lek'] ?

Reply:
ach bo ja skopiowałem ten formularz, z tego mojego starego projektu i tam właśnie wyświetlałem informacje o błędach za pomocą zmiennych sesyjnych. Wywalić to?